### PR TITLE
ci: Commit translations and create PR manually

### PR DESCRIPTION
**Pull request by kjarosh**
The Crowdin action does not allow adding custom modifications after downloading the translations and before creating the PR. Created the PR manually we can now modify files before creating the PR.

Additionally this PR reformats `crowdin.yml`.

This is a first step of automating localization of metainfo & .desktop file.

### CODE REVIEW

File: .github/workflows/download_translations.yml [hunk 1]
```diff
@@ -23,16 +23,29 @@ jobs:
...
           CROWDIN_PERSONAL_TOKEN: ${{ secrets.CROWDIN_PERSONAL_TOKEN }}
+
+      - name: Commit & push
+        run: |
+          git config user.name "RuffleBuild"
+          git config user.email "ruffle@ruffle.rs"
+          git checkout -b l10n_crowdin_translations
+          git add -A
+          git commit -m 'chore: Update translations from Crowdin'
+          git push --set-upstream origin l10n_crowdin_translations --force
```
Comment (torokati44) [inline .github/workflows/download_translations.yml:38]: Won't this command also require some kind of authentication/authorization; like the next via `$GITHUB_TOKEN`? :thinking:

Comment (kjarosh) [reply .github/workflows/download_translations.yml:38]: It's possible I messed something up, I'll replace that with the `ad-m/github-push-action` action that is used in other places and test

Comment (kjarosh) [reply .github/workflows/download_translations.yml]: done

**Review by torokati44 — APPROVED** [review]
I guess ... let's see?

**Comment by kjarosh** [comment]
It's ready to be merged now @torokati44

**Pull request merged**